docs/requirements/ics10-requirements.md

```diff
@@ -1,98 +1,94 @@
 
 # Functional requirements
 
-<!-- They should describe as completely as necessary the system's behaviors under various conditions. They describe what the engineers must implement to enable users to accomplish their tasks (user requirements), thereby satisfying the business requirements. Software engineers don't implement business requirements or user requirements. They implement functional requirements, specific bits of system behavior. Each requirement should be uniquely identified with a meaningful tag. -->
+## Assumptions
 
-The scope of this feature is to allow any implemention written in WASM to be compliant with the interface expressed
-in [02-client ClientState interface](../../modules/core/exported/client.go).
+1. This feature expects the [02-client refactor completed](https://github.com/cosmos/ibc-go/milestone/16), which is enabled in ibc-go v7.
 
-## Assumptions and dependencies
+## Features
 
-<!-- List any assumed factors that could affect the requirements. The project could be affected if these assumptions are incorrect, are not shared, or change. Also identify any dependencies the project has on external factors. -->
+### 1 - Configuration
 
-This feature expects the [02-client refactor completed](https://github.com/cosmos/ibc-go/milestone/16), which is enabled in `ibc-go v7`.
+| ID | Description | Verification | Status | 
+| -- | ----------- | ------------ | ------ | 
+| 1.01 | To enable the usage of the Wasm client module, chains must add update the `AllowedClients` parameter in the 02-client submodule. | TBD | Drafted |
+| 1.02 | The genesis state of the Wasm client module consists of the code ID and bytecode for each light client Wasm contract. | TBD | `Drafted` |
+| 1.03 | A chain shall have the ability to export the Wasm client module genesis state.	| TBD | `Drafted` |
+| 1.04 | A chain shall have the ability to initialize the Wasm client module genesis state. | TBD | `Drafted` |
 
-## Features
+### 2 - Initiation
 
-<!-- Use a table like the following for the requirements:
 | ID | Description | Verification | Status | 
-| -- | ----------- | ------------ | ------ | 
--->
+| -- | ----------- | ------------ | ------ |
+| 2.01 | Users must submit a governance proposal to store a light client implementation compiled in Wasm bytecode. | TBD | `Drafted` |
+| 2.02 | Once a light client Wasm contract has been stored, every light client will be created with a new instance of the contract. | TBD | `Drafted` |
+| 2.03 | The bytecode for each light client Wasm contract is stored in a client-prefixed store indexed by the hash of the bytecode. | TBD | `Drafted` |
+| 2.04 | The size in bytes of bytecode of the light client Wasm contract must be > 0 and <= 3 MiB. | TBD | `Drafted` |
+
```

Comment:
Should there be more information of the functional requirements wrt updating the client and consensus state or is it fine to have it expressed it must fit with the -2 client interfaces? I'm wondering if it would be good to split it up a bit more as there will be some acceptance tests linked for this functionality